python/Ganga/Runtime/bootstrap.py

```diff
@@ -319,6 +319,8 @@ def file_opens(f, message):
     @staticmethod
     def new_version():
         versions_filename = os.path.expanduser('~/.cache/Ganga/.used_versions')
+        if not os.path.exists(os.path.dirname(versions_filename)):
```

Comment:
according to the python website `makedirs` is recursive, `mkdir` is not, so this should be fine